crates/cli/src/bin/wasm-bindgen-test-runner/server.rs

```diff
@@ -218,6 +244,22 @@ pub fn spawn(
                 )
             };
             return set_isolate_origin_headers(Response::from_data("text/html", s));
+        } else if request.url() == "/__coverage/dump" {
+            fn internal_err(s: &str) -> Response {
+                log::error!("{s}");
+                let mut ret = Response::text(s);
+                ret.status_code = 500;
+                ret
+            }
+            let Some(profraw_path) = &coverage else {
```

Comment:
```suggestion
            let Some(profraw_path) = coverage else {
```

Reply:
```
error[E0507]: cannot move out of `coverage.0`, as `coverage` is a captured variable in an `Fn` closure
   --> crates/cli/src/bin/wasm-bindgen-test-runner/server.rs:254:38
    |
20  |     coverage: Option<PathBuf>,
    |     -------- captured outer variable
...
221 |     let srv = Server::new(addr, move |request| {
    |                                 -------------- captured by this `Fn` closure
...
254 |             let Some(profraw_path) = coverage else {
    |                      ------------    ^^^^^^^^
    |                      |
    |                      data moved here
    |                      move occurs because `profraw_path` has type `PathBuf`, which does not implement the `Copy` trait
    |
help: consider borrowing the pattern binding
    |
254 |             let Some(ref profraw_path) = coverage else {
```